setup.py

```diff
@@ -16,7 +16,7 @@
     install_requires=[
         'grpcio-tools==1.14.1',
         'jsonrpcclient==2.5.2',
-        'eth-typing<2', # workaround until next version after eth-abi v2.0.0-beta.1 is released
+        'eth-typing>=2',
```

Comment:
This line was added because of the issue #34 Fix issue with pycryptodome dependencies rised by new eth-hash v0.2.0

May be it is better just to remove this line at all?
Will eth-typing version 2 be got as dependency resolution in this case?